src/mocks/getMock.ts

```diff
@@ -0,0 +1,10 @@
+import { readFileSync } from "fs";
+export const getMock = (file: string, deserialize: boolean = true) => {
```

Comment:
Maybe deserialize flag isn't needed. If you want that can't you split it in two function : one for serialized output and one for deserialized ?